cmd/frakti/frakti.go

```diff
@@ -58,6 +58,8 @@ var (
 	enableAlternativeRuntime = pflag.Bool("enable-alternative-runtime", true, "Enable alternative runtime to handle OS containers, default is true")
 	cgroupDriver             = pflag.String("cgroup-driver", "cgroupfs", "Driver that the frakti uses to manipulate cgroups on the host. *SHOULD BE SAME AS* kubelet cgroup driver configuration.  Possible values: 'cgroupfs', 'systemd'")
 	rootDir                  = pflag.String("root-directory", "/var/lib/frakti", "Path to the frakti root directory")
+	defaultCPUNum            = pflag.Int("cpu-limit", 1, "Default CPU limit in number for HyperVM when cpu is not specified for the pod")
```

Comment:
I mean: `Default CPU in number for HyperVM when cpu limit is not specified for the pod`. Because there's cpu request in k8s which frakti do not care.